packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -30,7 +30,9 @@ export class CrossmintAASDK {
             let owner: SmartAccountSigner;
             if ((walletConfig.signer as FireblocksNCWSigner)?.type === "FIREBLOCKS_NCW") {
```

Comment:
make a function `isFireblockNCWSigner(signer): signer is FireblocksNCWSigner`

then u dont need to do `as FireblocksNCWSigner` 3 times